frontend/src/app/search.tsx

```diff
@@ -98,6 +125,60 @@ export default class Search extends Component {
     );
   }
 
+  private handleChange = (checked, event) => {
+    const target = event.target;
+    const value = target.type === 'checkbox' ? target.checked : target.value;
+    const name = target.name;
+    this.setState({ [name]: value }, ()=> {
+      if(checked === true){
+        this.setState({count: this.state.count+1}, () => {
+          if(this.state.count === 0){
+            this.setState({show: false})
+          }else{
+            this.setState({show: true})
+          }
+        })
+        this.tpaths.push(name);   
+        console.log('tpaths:'+this.tpaths)
+      }else{
+        this.setState({count: this.state.count-1}, () => {
+          if(this.state.count === 0){
+            this.setState({show: false})
+          }else{
+            this.setState({show: true})
+          }
+        })
+        this.tpaths.splice(this.tpaths.indexOf(name))
+        console.log('tpaths:'+this.tpaths)
+      }
+    });
+  };
+
+  delete = (event, keydata) => {
+    console.log('in the delete function')
+    for(var i=0;i<keydata.length;i++){
+      // console.log(keydata[i])
+      const formData = new FormData();
+      formData.append('module', keydata[i])
+      fetch('/content/'+keydata[i], {
```

Comment:
As we discussed this morning, this performs one POST per item deleted, but the goal is to perform one single POST with a multivalued parameter. I know you have fixed this in your latest code, so this is just a reminder for me to look at it in your next PR. :)

Reply:
This has been fixed in the current commit.